line/index.mjs

```diff
@@ -1,71 +1,70 @@
-import express from 'express'
-import * as utils from '../utilities/shared.mjs'
-import cors from 'cors'
-import { findLineById } from './line.mjs'
+import express from 'express';
```

Comment:
It looks like you have an auto formatter adding semicolons at the end of all the lines. That makes the diff harder to read and is not the style of the rest of the code. We only use ";" to when required to by the code (and then usually at the start of the offending line instead, for what it is worth).

Reply:
I have updated in the code